config/routes.rb

```diff
@@ -7,6 +7,7 @@
   resources :landing, only: [:index]
 
   get 'landing/hpi', to: 'landing#hpi'
+  get 'cookie', to: 'cookie#index'
```

Comment:
I think I'd go with a plural "cookies" over "cookie" here, but per your question `/privacy-policy` or `/cookies-policy` might be better.